code/rdb/rdbstandard.q

```diff
@@ -2,4 +2,11 @@
 .proc.getattributes:{`int`tables!(.rdb.getpartition[],();tables[])}
 
 \d .
-upd:{[t;x]};
+
+upd:$[`daily~.finspace.rollovermode;
+    .rdb.upd;
+    {[t;x]}];
+
+endofperiod:$[`daily~.finspace.rollovermode;
```

Comment:
On a slightly separate note, does it make more sense for this variable to be a bool rather than a sym? If it's only ever going to have two options it makes sense for it to be a bool. Are there potentially more options?

Reply:
I can't immediately see another potential option to include. From a user point of view however... I feel having the options of "daily" and "period" is more easily understood than an "on" and "off". Other than that yes it could be a bool.